canonical/ubuntu.com
# WD-14939 - update vanilla version and hr classes

## Done

Updated vanilla version to 4.16 and set correct classes on hr elements

## QA

- [demo link](https://canonical-com-1372.demos.haus/solutions/ubuntu-os)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the vanilla update didn't break anything on the page, that hr classes are correctly u

## Issue / Card
[WD-14939](https://warthogs.atlassian.net/browse/WD-14939)
